hazelcast/sql.py

```diff
@@ -848,13 +847,13 @@ def on_next_row(row_future):
     To get the number of rows updated by the query, use the
     :func:`update_count`. ::
 
-        update_count = client.sql.execute("SELECT ...").update_count().result()
+        update_count = client.sql.execute("UPDATE ...").result().update_count()
```

Comment:
I don't know but I copied this from the Java side. I assume they did it like this because currently, we don't have any queries that return update count